### ISSUE TITLE
Building of Full Stack Application scaffold doesn't work

### ISSUE THREAD

**Issue by ecomized** [webiny/webiny-js#1918]
### Version

5.14.0

### Operating System

Windows 10

### Browser

Firefox Nightly

### What steps will reproduce the bug?

- Fresh install of Webiny by following [Install Webiny](https://www.webiny.com/docs/tutorials/install-webiny)
- Running the Full Stack Application scaffold as descriped in [Create Custom Application/Getting Started](https://www.webiny.com/docs/tutorials/create-custom-application/getting-started#running-the-scaffold)

### What is the expected behavior?

![full-stack-app-scaffold-8cfe60ad0167cf7af055b3b752b42800](https://user-images.githubusercontent.com/66557654/134232852-c532aa52-18bf-41c9-bb30-8742df98d599.png)

### What do you see instead?

> 
> V@DESKTOP-ETQT28T MINGW64 ~/Documents/007_name-name/002_cms/full-example (master)
> $ yarn webiny scaffold
> ? Choose a scaffold: Full Stack Application
> ? Enter application name: Pinterest Clone
> ? Enter application description: This is the Pinterest Clone React application.
> ? Enter application path: pinterest-clone
> 
> The following operations will be performed on your behalf:
> - a new GraphQL API will be created in pinterest-clone\api
> - a new React application will be created in pinterest-clone\app
> - the list of workspaces will be updated in the root package.json file
> ? Are you sure you want to continue? Yes
> 
> ✔ New GraphQL API created in pinterest-clone\api.
> ✔ The list of workspaces in the root package.json file updated.
> ✔ Finalized.
> 
> ✔ New React Application created in pinterest-clone\app.
> ✔ The list of workspaces in the root package.json file updated.
> ✔ Finalized.
> 
> A Simple GraphQL API Example
> An example GraphQL API interaction can be included in the created React application code.
> ? Would you like to include it? No
> 
> Initial GraphQL API Deployment
> To begin developing, the new GraphQL API needs to be deployed.
> ? Do you want to deploy it now? Yes
> 
> Running yarn webiny deploy pinterest-clone\api command...
> 
> webiny success: Successfully loaded environment variables from .env.
> webiny info: Could not load environment variables from .env.dev.
> webiny success: Successfully loaded environment variables from .env.
> webiny info: Could not load environment variables from .env.dev.
> webiny info: Running 'build' in 1 packages...
> pinterest-clone-api: Usage Error: The nearest package directory (C:\Users\username\Documents\007_name-name\002_cms\full-example\pinterest-clone\api\code\graphql) doesn't seem to be part of the project declared in C:\Users\username\Documents\007_name-name\002_cms\full-example.
> - If the project directory is right, it might be that you forgot to list pinterest-clone/api/code/graphql as a workspace.
> - If it isn't, it's likely because you have a yarn.lock or package.json file there, confusing the project root detection.
> $ yarn run [--inspect] [--inspect-brk] <scriptName> ...
> webiny error: Script failed in package pinterest-clone-api!
> webiny error: Command failed with exit code 1: yarn webiny workspaces run build --folder C:/Users/username/Documents/007_name-name/002_cms/full-example/pinterest-clone/api --env dev --debug false
> webiny debug:  Error: Command failed with exit code 1: yarn webiny workspaces run build --folder C:/Users/username/Documents/007_name-name/002_cms/full-example/pinterest-clone/api --env dev --debug false
>     at makeError (C:\Users\username\Documents\007_name-name\002_cms\full-example\node_modules\@webiny\cli-plugin-deploy-pulumi\node_modules\execa\lib\error.js:60:11)
>     at handlePromise (C:\Users\username\Documents\007_name-name\002_cms\full-example\node_modules\@webiny\cli-plugin-deploy-pulumi\node_modules\execa\index.js:118:26)
>     at processTicksAndRejections (internal/process/task_queues.js:97:5)
>     at module.exports (C:\Users\username\Documents\007_name-name\002_cms\full-example\node_modules\@webiny\cli-plugin-deploy-pulumi\commands\deploy.js:38:9)
>     at Object.handler (C:\Users\username\Documents\007_name-name\002_cms\full-example\node_modules\@webiny\cli-plugin-deploy-pulumi\commands\index.js:47:21) {
>   shortMessage: 'Command failed with exit code 1: yarn webiny workspaces run build --folder C:/Users/username/Documents/007_name-name/002_cms/full-example/pinterest-clone/api --env dev --debug false',
>   command: 'yarn webiny workspaces run build --folder C:/Users/username/Documents/007_name-name/002_cms/full-example/pinterest-clone/api --env dev --debug false',
>   escapedCommand: 'yarn webiny workspaces run build --folder "C:/Users/username/Documents/007_name-name/002_cms/full-example/pinterest-clone/api" --env dev --debug false',
>   exitCode: 1,
>   signal: undefined,
>   signalDescription: undefined,
>   stdout: undefined,
>   stderr: undefined,
>   failed: true,
>   timedOut: false,
>   isCanceled: false,
>   killed: false
> }
> 
> Error: Command failed with exit code 1: yarn webiny deploy pinterest-clone\api --env dev --debug false
>     at makeError (C:\Users\username\Documents\007_name-name\002_cms\full-example\node_modules\@webiny\cli-plugin-scaffold-graphql-api\node_modules\execa\lib\error.js:60:11)
>     at handlePromise (C:\Users\username\Documents\007_name-name\002_cms\full-example\node_modules\@webiny\cli-plugin-scaffold-graphql-api\node_modules\execa\index.js:118:26)
>     at processTicksAndRejections (internal/process/task_queues.js:97:5)
>     at Object.onSuccess (C:\Users\username\Documents\007_name-name\002_cms\full-example\node_modules\@webiny\src\index.ts:237:17)
>     at scaffold (C:\Users\username\Documents\007_name-name\002_cms\full-example\node_modules\@webiny\src\scaffold.ts:68:13) {
>   shortMessage: 'Command failed with exit code 1: yarn webiny deploy pinterest-clone\\api --env dev --debug false',
>   command: 'yarn webiny deploy pinterest-clone\\api --env dev --debug false',
>   escapedCommand: 'yarn webiny deploy "pinterest-clone\\api" --env dev --debug false',
>   exitCode: 1,
>   signal: undefined,
>   signalDescription: undefined,
>   stdout: undefined,
>   stderr: undefined,
>   failed: true,
>   timedOut: false,
>   isCanceled: false,
>   killed: false
> }

**Comment by adrians5j** [comment]
Thanks for the detailed report.

Question 1 - in which folder is your project located?
Question 1.1 - I'm confused by the C:\Users\private-folder\full-example\pinterest-clone\api\code\graphql path, why is the `full-example` in it? 🤔 

Question 2 - did you run the webiny scaffold command from project root?

Thanks!

**Comment by ecomized** [comment]
Thank you for your fast response!

> 
> 
> Thanks for the detailed report.
> 
> Question 1 - in which folder is your project located?

***C:\Users\username\Documents\007_name-name\002_cms\full-example\***

> Question 1.1 - I'm confused by the C:\Users\private-folder\full-example\pinterest-clone\api\code\graphql path, why is the `full-example` in it? 🤔

**I just created this to test your new tutorial on how to create a pinterst clone. There you have named the project itself on [github ](https://github.com/webiny/webiny-examples/tree/master/create-custom-application/full-example/pinterest-clone)that way. So I did the same.**

**I have also cleaned up the folder path a little, due to private data. I have now once again adjusted a little above, so that it more resembles the original.**

> 
> Question 2 - did you run the webiny scaffold command from project root?

**Yes, I checked and I did.**

**Comment by adrians5j** [comment]
The project is not called `full-example`, it's just the name of the folder.

Just to confirm, you didn't do any copy/pasting from there right? So for example copying the full-example folder into your Webiny project?

You simply created a new Webiny project and ran the Full Stack Application scaffold (via `yarn webiny scaffold`), like described in the docs right?

Note that the `pinterest-clone` folder, located in the `full-example` folder is what you should also end up with, once the Full Stack Application has completed. For example, `C:/Users/my-username/new-webiny-project/pinterest-clone`.

One other thing, can you tell me what you have in your root `package.json`, in the `workspaces` property?

**Comment by ecomized** [comment]
> 
> 
> The project is not called `full-example`, it's just the name of the folder.
> 
> Just to confirm, you didn't do any copy/pasting from there right? So for example copying the full-example folder into your Webiny project?
> 

No, I did not.

> You simply created a new Webiny project and ran the Full Stack Application scaffold (via `yarn webiny scaffold`), like described in the docs right?
> 

Yes, I did. 

> Note that the `pinterest-clone` folder, located in the `full-example` folder is what you should also end up with, once the Full Stack Application has completed. For example, `C:/Users/my-username/new-webiny-project/pinterest-clone`.
> 
Yes, I agree but still got the error. 

> One other thing, can you tell me what you have in your root `package.json`, in the `workspaces` property?

	"workspaces": {
		"packages": [
			"apps/admin/code",
			"apps/website/code",
			"apps/theme",
			"api/code/dynamoToElastic",
			"api/code/fileManager/*",
			"api/code/graphql",
			"api/code/headlessCMS",
			"api/code/pageBuilder/*",
			"api/code/prerenderingService/render",
			"api/code/prerenderingService/flush",
			"api/code/prerenderingService/queue/*",
			"pinterest-clone\\api/code/graphql",
			"pinterest-clone\\app/code"
		]
	},

**Comment by adrians5j** [comment]
@ecomized We believe we've managed to reproduce the issue here. Notice the backwards slashes in the:

```
"pinterest-clone\\api/code/graphql",
"pinterest-clone\\app/code"
```

Try replacing them with `/`, then run `yarn`, and finally try to re-deploy with `yarn webiny deploy pinterest-clone/api --env dev`. 

We'll release a fix for this in the upcoming 5.16 (beta hopefully this week).
